packages/angular/projects/ui-angular/src/lib/primitives/amplify-input/amplify-input.component.ts

```diff
@@ -26,9 +30,25 @@ export class AmplifyInputComponent {
   @Input() initialValue = '';
   @Input() disabled = false;
   @Input() autocomplete = '';
+  public defaultCountryCode: string;
+  public countryDialCodes = countryDialCodes;
 
   constructor(private stateMachine: StateMachineService) {}
 
+  ngOnInit(): void {
+    const state = this.stateMachine.authState;
+    const { country_code }: ActorContextWithForms = getActorContext(state);
+    this.defaultCountryCode = country_code;
+
+    // TODO: consider better default handling mechanisms across frameworks
+    if (this.isTelInput()) {
+      this.stateMachine.send({
+        type: 'CHANGE',
+        data: { name: 'country_code', value: country_code },
+      });
+    }
+  }
```

Comment:
Dittoing the implementation from React: https://github.com/aws-amplify/amplify-ui/blob/36ce52646f455e6a3f2d7e9befb4f808efc1061b/packages/react/src/components/Authenticator/shared/UserNameAlias.tsx#L30-L36 We're looking to improve this side effect pattern in a separate pr (Issue #404, [Asana](https://app.asana.com/0/1200334286823427/1201022646170709)). Leaving this as now so I can get started on parity styling 👍